Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -756,7 +822,7 @@ HRESULT _DWriteCreateFontFaceWithName(CFStringRef name, IDWriteFontFace** outFon
     DWRITE_FONT_STRETCH stretch = DWRITE_FONT_STRETCH_NORMAL;
     DWRITE_FONT_STYLE style = DWRITE_FONT_STYLE_NORMAL;
 
-    __InitDWriteFontPropertiesFromName(name, &weight, &stretch, &style);
+    __InitDWriteFontPropertiesFromName(name, &weight, &stretch, &style, nullptr);
 
     CFStringRef familyName = _DWriteGetFamilyNameForFontName(name);
```

Comment:
whoops, didn't catch that

---
In reply to: [79946189](https://github.com/Microsoft/WinObjC/pull/1021#discussion_r79946189) [](ancestors = 79946189)